File: server/TWS Admin/Customer/Models/AccountIdentityModel.cs
using Foundation.Contracts.Modelling.Bases;

namespace Customer;

public class AccountIdentityModel
    : BModel {
    public string Identity {get; private set;} 
    public byte[] Password {get; private set;}

    public AccountIdentityModel(string identity, byte[] password) {
        Identity = identity;
        Password = password;
    }
}

File: client/tws_main/lib/domain/repositories/tws_administration/models/account_identity_model.dart
import 'dart:convert';
import 'dart:typed_data';

import 'package:cosmos_foundation/contracts/interfaces/i_model.dart';

typedef JSON = Map<String, dynamic>;

class AccountIdentityModel implements IModel {
  final String identity;
  final Uint8List password;

  const AccountIdentityModel(this.identity, this.password);
  factory AccountIdentityModel.fromJson(JSON json) {
    return AccountIdentityModel(json['Identity'], json['Password']);
  }

  @override
  JSON toJson() {
    return <String, dynamic>{
      'Identity': identity,
      'Password': base64.encode(password),
    };
  }
}
